presto-main/src/main/java/com/facebook/presto/server/protocol/QueuedStatementResource.java

```diff
@@ -110,14 +111,18 @@
     private final ConcurrentMap<QueryId, Query> queries = new ConcurrentHashMap<>();
     private final ScheduledExecutorService queryPurger = newSingleThreadScheduledExecutor(threadsNamed("dispatch-query-purger"));
 
+    private final SqlParserOptions sqlParserOptions;
+
     @Inject
     public QueuedStatementResource(
             DispatchManager dispatchManager,
             DispatchExecutor executor,
-            LocalQueryProvider queryResultsProvider)
+            LocalQueryProvider queryResultsProvider,
+            SqlParserOptions sqlParserOptions)
     {
         this.dispatchManager = requireNonNull(dispatchManager, "dispatchManager is null");
         this.queryResultsProvider = queryResultsProvider;
+        this.sqlParserOptions = sqlParserOptions;
```

Comment:
use requireNonNull, similar to how dispatchManager is set above.  (Not sure why there's a second requireNonNull for dispatch manager below.  that could be removed if you'd like.)